conduce/api.py

```diff
@@ -229,6 +229,7 @@ def wait_for_job(job_id, **kwargs):
                 if kwargs.get('cli'):
                     print("Job status check failed for {}:".format(job_id), e.response.reason)
                     print("Will retry after sleep period.")
+                    time.sleep(0.5)
```

Comment:
I think this change is not right.  I want to move the sleep to after the execution of make_get_request.  So I guess I need to add another sleep at the end of the try block.